packages/flutter/test/material/text_field_test.dart

```diff
@@ -7398,4 +7398,37 @@ void main() {
     await tester.pump();
     expect(focusNode3.hasPrimaryFocus, isTrue);
   });
+
+  testWidgets('test empty widget when the buildCounter returns null',
```

Comment:
Improved wording:

"A buildCounter that returns null doesn't affect the size of the TextField"

Reply:
Done.